modules/test/playwright/tests/object-web/utils/getRandomObjectField.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {getRandomInt} from '../../../utils/getRandomInt';
+
+interface Props {
+	fieldsQuantity: number;
+}
+
+export default function getRandomObjectField({
```

Comment:
Hey @vngoliveira, since we are only using text business type, what do you think about add this info here? 

So we can use `getRandomObjectFieldText`, if we decide to make this function more complex by returning different business types. We can go back and use the more generic name.